src/popup-generation/FieldInputRow.js

```diff
@@ -53,11 +49,8 @@ export const FieldInputRow = ({index, fieldInputValue, onChange, onDeleteField})
 				className="field-type"
 				onChange={(fieldType) => onChange({...fieldInputValue, fieldType})}
 			>
-				<Select.Option value={FIELD_TYPES.number}>{FIELD_TYPES.number}</Select.Option>
-				<Select.Option value={FIELD_TYPES.text}>{FIELD_TYPES.text}</Select.Option>
-				<Select.Option value={FIELD_TYPES.zillowLink}>{FIELD_TYPES.zillowLink}</Select.Option>
-				<Select.Option value={FIELD_TYPES.wikipediaLink}>{FIELD_TYPES.wikipediaLink}</Select.Option>
-				<Select.Option value={FIELD_TYPES.subSectionHeader}>{FIELD_TYPES.subSectionHeader}</Select.Option>
+				{Object.values(FIELD_TYPES).map(fieldType => <Select.Option key={fieldType} value={fieldType}>{fieldType}</Select.Option>)}
+				
```

Comment:
(ignore this, this was just a code improvement to generate them programmatically vs hard coding each value)